lib/browser/browser.js

```diff
@@ -31,14 +31,16 @@ function OktaAuthBuilder(args) {
   var sdk = this;
 
   var url = builderUtil.getValidUrl(args);
+  // OKTA-242989: support for grantType will be removed in 3.0 
+  var usePKCE = args.pkce || args.grantType === 'authorization_code';
```

Comment:
This change overrides my configuration when I have `pkce: false` and `grantType: 'authorization_code'`.

Do you 'have' to use pkce with an auth_code grant?

There are a few places in the code that do this type of boolean coalesce operation that override an explicit parameter that is already set to `false`

Reply:
The `grantType` parameter was added specifically for PKCE, it has no other function. We have deprecated it because of the confusion. To perform "standard" `authorization_code` flow, you should set the `responseType` to `['code']` and do not use the `grantType` or `pkce` option.